ex1/functions.py

```diff
@@ -90,26 +192,127 @@ def f():
 # Define a function named args_to_dict that takes three arguments and returns a
 # dictionary with the position of the argument as the key (starting at 0) and
 # the argument as the value.
+def args_to_dict(a, b, c):
+    """
+    takes three arguments and returns a dictionary with the position of the
+    arguments as the key (starting at 0) and the argument as the value
+    """
+    return {0:a, 1:b, 2:c}
 
 # BONUS: Write a function named args_to_dict_general that does the same for any
 # number of arguments
+def args_to_dict_general(*args):
+    """
+    takes n arguments and returns a dictionary with the position of the
+    arguments as the key (starting at 0) and the argument as the value
+    """
+    i = 0
+    abc = {}
+    for arg in args:
+        abc[i] = arg
+        i += 1
+    return abc
 
 # Define a function named lists_to_dict that takes two lists of equal lenght
 # named keys and values and builds a dictionary out of them.
+def lists_to_dict(list1, list2):
+    """
+    builds a dictionary of two lists of equal length
+    Parameters
+    ----------
+    list1: list
+        first content (keys)
+    list2: list
+        second content (values)
+    Returns
+    ----------
+    abc: dict
+        of the two lists
+    """
+    i = 0   
+    abc = {}
+    for vals in list1:
```

Comment:
If you have two (or more) lists of equal lengths use the `zip` function. It is less code and less error prone than implementing the counting code. See #20